base/Dockerfile.rhel

```
@@ -18,5 +18,9 @@ RUN INSTALL_PKGS=" \
     yum install -y --setopt=tsflags=nodocs --setopt=install_weak_deps=False ${INSTALL_PKGS} && \
     ( test -e /usr/bin/python ||  alternatives --set python /usr/bin/python3 ) && \
     yum clean all && rm -rf /var/cache/*
+
+# Enable x509 common name matching for golang 1.15 and beyond.
+ENV GODEBUG=${GODEBUG}:x509ignoreCN=0
```

```suggestion
ENV GODEBUG=x509ignoreCN=0
```
Sorry, after looking itno this and testing the colon is an aspect of bash path defintion, golang expects these variables to be comma separated key=value, given this is base layer i'd just keep things as simple as possible and revert back to how you had it.